### DISCUSSION TITLE
docs: How to updated related entity id?

### DISCUSSION THREAD

**Discussion by cloudever** [mikro-orm/mikro-orm#2870] [Q&A]
Suppose there are 2 entities like Author and Book.

Simplified code looks like below:
```js
@Entity()
class Author {
  @PrimaryKey()
  id: number

  @Property()
  name: string
}

@Entity()
class Book {
  @PrimaryKey()
  id: number

  @Property()
  name: string
  
  @ManyToOne(() => Author)
  author: Author
}
```

Imagine that frontend sends a payload for updating Book and contains `authorId` field for updating author field, for example
```js
{ name: 'new name', authorId: 2 }
```

I found similar issue for `type-orm`
https://stackoverflow.com/questions/67212239/typeorm-update-entity-column-with-relation-joincolumn-via-repository

Using similar approach to `mikro-orm` gives double `author_id` field in migration generator and schema generator.

How to update author (or `authorId` field) using a minimum code and keep type safety?

**Comment by ttsirkia** [comment]
Are you updating the name of the book or the author? Have you checked this page? https://mikro-orm.io/docs/entity-helper
